components/ide-metrics/pkg/server/server.go

```diff
@@ -232,12 +232,11 @@ func (s *IDEMetricsServer) Start() error {
 	}))
 
 	c := cors.New(cors.Options{
+		AllowCredentials: true,
```

Comment:
should it be deployed before?

Reply:
both `ide-metrics` and `supervisor` is belong to us, they will deploy together.

fetch in openvscode-server is wrapped by catch, will only affect red(failed) requests in DevTool Network section and console